docs/eventing/accessing-traces.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Accessing CloudEvent Traces"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 15
+type: "docs"
+---
+
+Depending on the request tracing tool that you have installed on your Knative
+Eventing cluster, see the corresponding section for details about how to
+visualize and trace your requests.
+
+## Installation
```

Comment:
Since the doc  /serving/installing-logging-metrics-traces.md is under the Serving branch, can we clarify by adding an example to that its clear that you dont need to install Serving too? Long term we should move the observability content into its own branch in the website.

Maybe something like:

For example if you want to install Prometheus and Grafana, you can run:
`kubectl apply --filename https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-metrics-prometheus.yaml`

Reply:
How about something like:

Knative Eventing uses the same tracing installation as Knative Service. Follow Knative Serving's [installation instructions](./../serving/installing-logging-metrics-traces.md) to set up the necessary components. Note that you do not need to install Knative Serving itself. Only Zipkin or Jaeger and its dependencies (e.g. Elasticsearch).

Reply:
How about the edit i added below?

Reply:
(1) Also, would it help if the specific install files (.yamls) were listed under Eventing in the "custom install" file too: https://github.com/knative/docs/blob/master/docs/install/Knative-custom-install.md#choosing-knative-installation-files

(2) I guess whats unclear is which .yaml file do you install to get Elasticsearch? For example, for Serving we say to install the ELK stack (monitoring-logs-elasticsearch.yaml) to enable Logging. 
(2a) Is that the file that should be installed with Eventing? 
(2b) And do you get the same Logging features in Eventing?

Reply:
(1) All of the files that would be listed are already in  "Serving Component and Observability Plugins". Maybe it is worthwhile to split those into two distinct heading, 'Serving Component' and 'Observability Plugins'?

(2a) Yes, you install monitoring-logs-elasticsearch.yaml and either monitoring-tracing-zipkin.yaml or monitoring-tracing-jaeger.yaml (I think the in-mem versions might work without elasticsearch, but I haven't tried).
(2b) I'm not sure, I've never tried them.

Reply:
I think i will just move all the observability plugins into their own section for now.

Reply:
PR for separate Observability section: https://github.com/knative/docs/pull/1718